src/main/java/org/fundacionjala/coding/magali/EANValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.coding.magali;
+
+public final class EANValidator {
+    private EANValidator() {
+    }
+
+    public static boolean validate(final String eanCode) {
+        final int ten = 10;
+        final int multiplier = 3;
+        String[] numberArray = eanCode.split("");
+        int sum = 0;
+        for (int i = 0; i < numberArray.length - 1; i++) {
+            int number = Integer.parseInt(numberArray[i]);
+            if (i % 2 == 0) {
+                sum += number;
+            } else {
+                sum += number * multiplier;
+            }
+        }
+        int checksum = 0;
+        if (sum % ten != 0) {
```

Comment:
You can use a ternary operator here
int checksum = sum % ten != 0 ? ten - (sum % ten) : 0;

Reply:
Carlos, applied the changes you suggested.